client/src/public/ts/opponent.ts

```
@@ -71,45 +90,33 @@ class Opponent {
-    this.stageItems.textSprites.name.text = `${data.ownerName}`;
+    // text
+    this.nameText.text = data.ownerName;
+
+    const health = data.baseHealth;
+    const combo = Math.max(data.combo, 0);
+    const stock = Math.max(data.receivedEnemiesStock, 0);
+
+    this.statsText.text = `♥${health} 🡕${combo} 🞎${stock}`;
+
+    // enemies
     for (let enemy of data.enemies) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'enemy' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)